README.md

```diff
@@ -661,36 +718,57 @@ convert trivy2hdf         Translate a Trivy-derived AWS Security Finding
     AWS_REGION=us-east-1 AWS_ACCOUNT_ID=123456789012 trivy image --no-progress --format template --template "@/absolute_path_to/git_clone_of/trivy/contrib/asff.tpl" -o trivy_asff.json golang:1.12-alpine
 
   EXAMPLES
-    saf convert trivy2hdf -i trivy_asff.json -o output-hdf-name.json
+    $ saf convert trivy2hdf -i trivy-asff.json -o output-folder
 ```
 
+##### Twistlock to HDF
+```
+convert twistlock2hdf         Translate a Twistlock CLI output file into an HDF results set
+
+  USAGE
+    $ saf convert twistlock2hdf -i <twistlock-json> -o <hdf-scan-results-json>
+
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input Twistlock file
+    -o, --output=<value>  (required) Output HDF JSON File
+
+  EXAMPLES
+    $ saf convert twistlock2hdf -i twistlock.json -o output-hdf-name.json
+```
 
 ##### XCCDF Results to HDF
 NOTE: `xccdf_results2hdf` only supports native OpenSCAP output and SCC output.
 ```
-convert xccdf_results2hdf    Translate a SCAP client XCCDF-Results XML report to
-                             HDF format Json be viewed on Heimdall
-  OPTIONS
-    -i, --input=input          Input XCCDF Results XML File
-    -o, --output=output        Output HDF JSON File
+convert xccdf_results2hdf     Translate a SCAP client XCCDF-Results XML report
+                              to HDF format Json be viewed on Heimdall
+  USAGE
+    $ saf convert xccdf_results2hdf -i <xccdf-results-xml> -o <hdf-scan-results-json> [-h]
 
-  EXAMPLES
-    saf convert xccdf_results2hdf -i results-xccdf.xml -o output-hdf-name.json
+  FLAGS
+    -h, --help            Show CLI help.
+    -i, --input=<value>   (required) Input XCCDF Results XML File
+    -o, --output=<value>  (required) Output HDF JSON File
 
+  EXAMPLES
+    $ saf convert xccdf_results2hdf -i results-xccdf.xml -o output-hdf-name.json
 ```
 
 ##### OWASP ZAP to HDF
-
 ```
-convert zap2hdf              Translate a OWASP ZAP results JSON to HDF format Json
-                             be viewed on Heimdall
-  OPTIONS
-    -i, --input=input          Input OWASP ZAP Results JSON File
-    -n, --name=name            Target Site Name
-    -o, --output=output        Output HDF JSON File
+convert zap2hdf               Translate a OWASP ZAP results JSON to HDF format Json
```

Comment:
spell out hdf here like we do everywhere else.  also json should always be all caps or all lowercase.